lib/routes.js

```diff
@@ -742,17 +813,6 @@ module.exports = function(self, options) {
       function getRelated(callback) {
         return self.getRelated([ doc ], function(err, results) {
           related = results;
-          // Discard unless something is actually joined
```

Comment:
This logic is now in self.getRelated where it belongs. Didn't want to start replicating it everywhere.